src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div data-qa="card" class="card">
+      <img 
+        class="card__img" 
+        src="images/imac.jpeg" 
+        alt="imac"
+      >
+      <p class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__pcode">
+        Product code: 195434
+      </p>
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews-count">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+        <p class="card__price-number">
+          $2,199
+        </p>
+      </div>
+
+      <a 
+        class="card__btn" 
+        data-qa="hover" 
+        href="#"
+      >
+        buy
+      </a>
+      
+    </div>
```

Comment:
add spaces between multiple siblings, not between parents and children
```suggestion
      <a 
        class="card__btn" 
        data-qa="hover" 
        href="#"
      >
        buy
      </a>
    </div>
```